Dockerfile

```diff
@@ -1,33 +1,48 @@
-FROM python:3.6-alpine3.10 as base
+FROM python:3.7-alpine3.12 as base
```

Comment:
let's go ahead and update this to 3.8 while we're at it
```suggestion
FROM python:3.8-alpine3.12 as base
```

Reply:
Good idea.